locales/en.default.schema.json

```diff
@@ -1776,15 +1776,45 @@
         "show_background_image": {
```

Comment:
@wiktoriaswiecicka 

Currently the success message is always aligned to the left (even when everything was defaulted to be centered) . Should it follow the text alignment as well set by the merchant? What do you think? 🤔 

https://screenshot.click/18-31-7mcyp-2eldu.png

Reply:
It should - I will update this in the `add-image-banner-settings` branch